Mathlib/Combinatorics/SetFamily/Compression/Down.lean

```diff
@@ -164,8 +164,10 @@ lemma image_insert_memberSubfamily (𝒜 : Finset (Finset α)) (a : α) :
 it suffices to prove it for
 * the empty finset family.
 * the finset family which only contains the empty finset.
-* `𝒜 ∪ {s ∪ {a} | s ∈ ℬ}` assuming the property for `𝒜` and `ℬ`, where `a` is an element of the
+* `ℬ ∪ {s ∪ {a} | s ∈ 𝒞}` assuming the property for `ℬ` and `𝒞`, where `a` is an element of the
   ground type and `𝒜` and `ℬ` are families of finsets not containing `a`.
+  Note that instead of giving `ℬ` and `𝒞`, the induction principle gives you
```

Comment:
Mentioning the name of the relevant hypothesis would be nice, especially as this is what the user would type after `induction`:
```suggestion
  Note that instead of giving `ℬ` and `𝒞`, the `subfamily` case gives you
```